homeassistant/components/wallbox/__init__.py

```diff
@@ -23,7 +23,27 @@
 UPDATE_INTERVAL = 30
 
 
-class WallboxCoordinator(DataUpdateCoordinator[Dict[str, Any]]):
+class WallboxData(TypedDict):
+    """Data structure for returned Wallbox data.
+
+    Assume 'total=True' although fields could be missing.
+    Entities are only added if fields are present.
+    """
+
+    depot_price: float
+    status_description: str
+    charging_power: int  # maybe float?
+    max_available_power: int
+    charging_speed: int  # maybe float?
+    added_range: int  # maybe float?
+    added_energy: float
+    cost: int  # maybe float?
```

Comment:
I would make `cost` and `charging speed` a float. Other types seems to be correct.